Source/AssetRipper.Processing/AnimatorControllers/VirtualAnimationFactory.cs

```diff
@@ -130,18 +130,28 @@ private static IChildMotion AddAndInitializeNewChild(this IBlendTree tree, Proce
 			IMotion? motion = state.CreateMotion(file, controller, childNodeIndex);
 			childMotion.Motion.SetAsset(tree.Collection, motion);
 
+			IBlendTreeNodeConstant targetNode = treeConstant.NodeArray[childNodeIndex].Data;
```

Comment:
I would prefer this be named `childNode` to align with the naming for the other local variables.